CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ------
 
 **ENHANCEMENTS**
+- Add the configuration parameter `DefaultUserHome` under `DeploymentSettings` to allow users to move the default user's home directory to `/local/home` instead of `/home` (default).
```

Comment:
*Minor*
If you want to specify the section where the parameter is placed (good practice), we should write "configuration parameter `DeploymentSettings/DefaultUserHome`"